### PR TITLE
fix(editor): Fix ts and eslint errors in various components (no-changelog)

### PR DESCRIPTION
**Pull request by tomi**
## Summary

- fix(editor): Fix ts and eslint errors

## Review / Merge checklist
- [ ] PR title and summary are descriptive. **Remember, the title automatically goes into the changelog. Use `(no-changelog)` otherwise.** ([conventions](https://github.com/n8n-io/n8n/blob/master/.github/pull_request_title_conventions.md))
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included.
   > A bug is not considered fixed, unless a test is added to prevent it from happening again.
   > A feature is not complete without tests.

### CODE REVIEW

File: packages/editor-ui/src/components/TagsContainer.vue [hunk 1]
```diff
@@ -135,9 +140,20 @@ export default defineComponent({
...
+	created() {
+		this.debouncedSetMaxWidth = debounce(this.setMaxWidth, 100);
+	},
+	mounted() {
+		this.setMaxWidth();
+		window.addEventListener('resize', this.debouncedSetMaxWidth);
+	},
+	beforeUnmount() {
+		window.removeEventListener('resize', this.debouncedSetMaxWidth);
+	},
```
Comment (tomi) [inline packages/editor-ui/src/components/TagsContainer.vue:152]: Reordered, no changes

File: packages/editor-ui/src/components/TimeAgo.vue [hunk 2]
```diff
@@ -17,20 +17,40 @@ export default defineComponent({
...
+			}
+
+			return text;
+		},
+		convertDate(): string {
+			const date = new Date(this.date);
+			const epoch = date.getTime();
+			return convertToHumanReadableDate(epoch);
+		},
+	},
```
Comment (tomi) [inline packages/editor-ui/src/components/TimeAgo.vue:46]: Moved

File: packages/editor-ui/src/components/TagsContainer.vue [hunk 3]
```diff
@@ -70,7 +70,22 @@ interface TagEl extends ITag {
...
 	components: { IntersectionObserver, IntersectionObserved },
-	props: ['tagIds', 'limit', 'clickable', 'responsive', 'hoverable'],
+	props: {
+		tagIds: {
+			type: Array as () => string[],
+			required: true,
+		},
+		limit: {
+			type: Number,
+			default: null,
```
Comment (MiloradFilipovic) [inline packages/editor-ui/src/components/TagsContainer.vue:80]: Nit: Should we expand the type to `Number | null` or should the default here be `DEFAULT_MAX_TAGS_LIMIT`?

**Review by MiloradFilipovic — DISMISSED** [review]
Looks good, left a minor comment but good to go 🚀

**Review by ivov — COMMENTED** [review]
`(no-changelog)` please, thanks for doing this!

**Comment by tomi** [comment]
> `(no-changelog)` please, thanks for doing this!

Thanks! Forgot to add it

File: packages/editor-ui/src/components/TagsContainer.vue (hunk 3 above)
Comment (tomi) [reply packages/editor-ui/src/components/TagsContainer.vue]: Good point. `DEFAULT_MAX_TAGS_LIMIT` makes more sense here

**Comment by tomi** [comment]
@MiloradFilipovic fixed the comment. Could you reapprove 🙏

**Review by MiloradFilipovic — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.45.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.45.0)

**Pull request merged**